local_planner/src/nodes/local_planner.h

```diff
@@ -186,6 +186,8 @@ class LocalPlanner {
 
  public:
 
+  double H_FOV_ = 59.0;
```

Comment:
usually only constants/defines are capitalized

Reply:
do you want me to decapitalize it? It was like that before, because it kinda is a constant that is only defined at the beginning but cannot make it const as I need to set it